bdb/info.c

```diff
@@ -1302,6 +1301,7 @@ void bdb_process_user_command(bdb_state_type *bdb_state, char *line, int lline,
         " del #          - remove node # from sanc list",
         " rem #          - force removal node # from cluster",
         " hostname #     - print hostname associated with node #", 
+        " nodenum #      - print nodenum associated with node name", 
```

Comment:
Maybe change '#' to '\<name\>' or something like that to make it clear that this expects a node name and not a number.

Reply:
yes, makes sense! will fix